src/tabs/discover/index.js

```diff
@@ -29,6 +29,7 @@ const EnhancedDiscover = withTheme(({ theme, ...props }) => ({
     headerTitleStyle: {
       color: theme.colors.text.primary,
     },
+    headerBackgroundColor: theme.colors.background.paper,
```

Comment:
maybe this is why the color changed above, we may not have this color defined

Reply:
yea we don't have these colors defined anywhere. not a big deal now but could create problems later.

Reply:
probably should drop in the colors to the theme file. also maybe do a grep search for any others in source code that may be undefined.

Reply:
![Screenshot 2020-04-27 16 06 03](https://user-images.githubusercontent.com/3229463/80415806-0a2c2980-88a1-11ea-8806-52a8c4968ccb.png)

So the theme somehow gets those values. Defaults from core maybe? 

When I added the `theme.colors.background.paper` value and ran the tests, nothing changed. Do you think it should have changed that test above?

Reply:
strange, yea I would have thought so